### ISSUE TITLE
Avatar component not focusable with keyboard

### ISSUE THREAD

**Issue by georgehrke** [nextcloud/nextcloud-vue#613]
Meaning that the contacts-menu is not also not accessible.

**Comment by skjnldsv** [comment]
:fixit: :D

**Comment by PVince81** [comment]
ideally we should move to the Actions button as it has everything needed for keyboard focus and moving through the items with arrows: https://github.com/nextcloud/nextcloud-vue/pull/1670

**Comment by PVince81** [comment]
WIP PR here: https://github.com/nextcloud/nextcloud-vue/pull/1854

but not sure if it can be finishable as the interaction in the popover is missing
